source/theme.c

```diff
@@ -1114,14 +1114,21 @@ static gboolean rofi_theme_get_image_inside(Property *p, const widget *widget,
       default:
         break;
       }
+#if 0
```

Comment:
Do we still need this commented out code?

Reply:
I just left it for future reference. If you want I can remove it.